pkg/gui/containers_panel.go

```diff
@@ -18,11 +18,11 @@ import (
 // list panel functions
 
 func (gui *Gui) getContainerContexts() []string {
-	return []string{"logs", "stats", "config", "top"}
+	return []string{"logs", "stats", "env", "config", "top"}
```

Comment:
could we also add this for the services panel?

Reply:
Added the `container-env` section for services panel